controller.go

```diff
@@ -106,6 +111,7 @@ func New() (NetworkController, error) {
 	if err := initDrivers(c); err != nil {
 		return nil, err
 	}
+	c.handler = newHTTPHandler(c)
```

Comment:
Makes sense. That way we can remove the rest handling API from NetworkController, northbound serving API should be separated from backend.
I actually started that way but got stuck into circular dependencies. 
I will try now again.